### ISSUE TITLE
Change the Video player to motion layout to make it a youtube like miniplayer

### ISSUE THREAD

**Issue by justdvnsh** [justdvnsh/AnimeClassroom#36]
We want to change the video player to make it a youtube like miniplayer, which can be slide up and down as and when required.

**Comment by Rishavgupta12345** [comment]
can i work on this issue ?

**Comment by justdvnsh** [comment]
@Rishavgupta12345 Yeah sure. We are so happy to have you here. Happy coding. Please star, the repo, if possible. It really helps a lot.
